### DISCUSSION TITLE
[Contribution] Updated Swedish translation

### DISCUSSION THREAD

**Discussion by Imerion** [bgrabitmap/lazpaint#589] [General]
Hi! I have updated the Swedish translation for the current dev version. A few strings were missing but now they are all added and the translation should be 100%.
[lazpaint_swedish_translation.zip](https://github.com/user-attachments/files/16913890/lazpaint_swedish_translation.zip)

**Comment by circular17** [comment]
Thank you very much. I've incorporated it into the dev branch.
